components/menu/src/flyout-menu/flyout-menu.js

```diff
@@ -61,6 +63,7 @@ FlyoutMenu.propTypes = {
     dataTest: PropTypes.string,
     /** Menu uses smaller dimensions */
     dense: PropTypes.bool,
+    isFocused: PropTypes.bool,
```

Comment:
I think it would be good to add these also to the typescript types in components/menu/types/index.d.ts